src/game/level/background.c

```diff
@@ -7,6 +7,9 @@
 #include "system/lt.h"
 #include "system/nth_alloc.h"
 #include "system/log.h"
+#ifdef WINDOWS_OS
+#include <stdio.h> // for EOF macro
```

Comment:
`stdio.h` exists on all of the platforms. Just include it without `#ifdef WINDOWS_OS`